src/data-dictionary/events/K8sContainerSample/aa_template.md

```diff
@@ -0,0 +1,9 @@
+---
+name: 
```

Comment:
missing a name

Reply:
@zuluecho9 this was just a template file. This has been removed now.